tensorflow/core/kernels/pad_op.cc

```diff
@@ -177,11 +193,17 @@ TF_CALL_POD_TYPES(REGISTER_KERNEL);
 namespace functor {
 #define DECLARE_GPU_SPEC(T, Dims)                                         \
   template <>                                                             \
-  void Pad<GPUDevice, T, Dims>::operator()(                               \
+  void Pad<GPUDevice, T, int32, Dims>::operator()(                               \
```

Comment:
please fix the alignment of the \ at the end of the line to match the other lines

Reply:
Thanks @rryan. The alignment has been fixed.